daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1104,36 +1099,11 @@ void IEEE1588Port::processEvent(Event e)
 			   system_time, local_system_freq_offset, sync_count,
 			   pdelay_count, port_state, asCapable );
 
-			if (!automotive_profile) {
-				/* If accelerated_sync is non-zero then start 16 ms sync
-				   timer, subtract 1, for last one start PDelay also */
-				if( _accelerated_sync_count > 0 ) {
-					clock->addEventTimerLocked
-					  ( this, SYNC_INTERVAL_TIMEOUT_EXPIRES, 8000000 );
-					    --_accelerated_sync_count;
-				} else {
-					syncDone();
-					if( _accelerated_sync_count == 0 ) {
-						--_accelerated_sync_count;
-					}
-					wait_time *= 1000; // to ns
-					wait_time =
-					  ((long long)
-					     (pow((double)2,getSyncInterval())*1000000000.0)) -
-					    wait_time;
-					wait_time = wait_time > EVENT_TIMER_GRANULARITY ? wait_time :
-					            EVENT_TIMER_GRANULARITY;
-					startSyncIntervalTimer(wait_time);
-				}
-			}
-			else {
-				// Automotive Profile
-				syncDone();
+			syncDone();
 
```

Comment:
The previous code subtracted the old `wait_time` from the new `wait_time`; as far as I can tell this sets the wait interval from the beginning of the Sync interval rather than from the point we've finished waiting for the hardware to give us a TX Timestamp for our first Sync.

I'm not sure whether you left this out intentionally to simplify things or if you missed it.  I'm not sure if it makes much difference correctness-wise.